### ISSUE TITLE
Alter TimeBlockView ToolTipText Display Functionality

### ISSUE THREAD

**Issue by William-Lake** [William-Lake/TimeTracker#18]
Currently, when viewing the custom ToolTipText for the TimeBlockView, it closes and opens whenever your mouse moves while hovering over the TimeBlockView. Instead, would like the following to happen when the mouse hovers:
- Open ToolTipText at specific Location
    - Thinking upper left corner is 10px up and 10px left of bottom right corner of TimeBlockView
- Leave it open until the mouse is no longer hovering over that TimeBlockView.

I think the code currently uses a Mouse Hovering listener. Might be a better idea to use a listener which checks for when the mouse enters and exits the TimeBlockView area.

**Comment by William-Lake** [comment]
[DefaultToolTip.setShift()](https://www.programcreek.com/java-api-examples/index.php?source_dir=eclipse.platform.ui-master/examples/org.eclipse.jface.snippets/Eclipse%20JFace%20Snippets/org/eclipse/jface/snippets/window/Snippet031TableStaticTooltip.java) might be useful here. May be able to pass it the new Point object using the parent [TimeBlockView](https://github.com/William-Lake/TimeTracker/blob/master/src/timetracker/ui/customobjects/controls/TimeBlockView.java)'s point created when calling computeSize(), and then adding 5 to both the x and y.

Will need to identify how often the new location is calculated/updated. Probably shouldn't happen every time the TimeBlockView.updateView() is called, but only when the ToolTip is displayed. 

Looks like [DefaultToolTip](https://help.eclipse.org/neon/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fjface%2Fwindow%2FDefaultToolTip.html) is inheriting the show() method from it's parent class. It would make sense if that is what's called to display the tooltip. May be able to @Override it and edit the Point here. This way the calculation is only performed when the ToolTip is going to be shown.

**Comment by William-Lake** [comment]
So now that Issue #16 has been addressed, the tooltip no longer needs to be displayed in the upper left of the TimeBlock. However, I found a pretty quick and easy way to persistently show the ToolTip while the mouse is hovering: [See the fields and the second constructor](http://help.eclipse.org/oxygen/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fjface%2Fwindow%2FToolTip.html).

Issue moving to testing.
